modules/configuring-scale-bounds-knative.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * serverless/configuring-knative-serving-autoscaling.adoc
+
+[id="configuring-scale-bounds-knative_{context}"]
+= Configuring scale bounds Knative Serving autoscaling
+
+The `minScale` and `maxScale` annotations can be used to configure the minimum and maximum number of pods that can serve applications.
+These annotations can be used to prevent cold starts or to help control computing costs.
+
+minScale:: If the `minScale` annotation is not set, pods will scale to zero (or to 1 if enable-scale-to-zero is false per the `ConfigMap`).
+
+maxScale:: If the `maxScale` annotation is not set, there will be no upper limit for the number of pods created.
+
+`minScale` and `maxScale` can be configured as follows in the revision template;
+
+[source,yaml]
+----
+spec:
+  template:
+    metadata:
+      autoscaling.knative.dev/minScale: "2"
+      autoscaling.knative.dev/maxScale: "10"
+----
+
+Using these annotations in the revision template will propagate this confguration to `PodAutoscaler` objects.
+`PodAutoscaler` objects are mutable and can be further modified later without modifying anything else in the Knative Serving system.
+
+----
```

Comment:
Should there be any sort of intro statement before this "oc edit" command? I just wasn't sure what the intention for the user to do this is.

Reply:
Good question - @markusthoemmes what does this command actually do, is it just to apply/propagate the config or do we need to talk about editing the podautoscaler in more detail?

Reply:
Let's ditch the flow for now. I allows you to edit the YAML of the podautoscaler on the fly, but we'd likely need to talk about what that means before we propose that.

Let's just not mention that you can edit the PodAutoscaler like that for now.